src/plots/gl2d/scene2d.js

```diff
@@ -522,6 +522,21 @@ proto.draw = function() {
                             glplot.pixelRatio
                 ];
 
+                // this needs to happen before the next block that deletes traceCoord data
+                // also it's important to copy, otherwise data is lost by the time event data is read
+                this.graphDiv.emit('plotly_hover', {
+                    points: [{
+                        trace: nextSelection.trace,
+                        dataCoord: nextSelection.dataCoord.slice(),
+                        traceCoord: nextSelection.traceCoord.slice(),
+                        textLabel: nextSelection.textLabel,
+                        color: nextSelection.color,
+                        name: nextSelection.name,
+                        hoverinfo: nextSelection.hoverinfo,
+                        screenCoord: nextSelection.screenCoord.slice()
```

Comment:
@etpinard just to clarify, you want to be as close to the data contents of the event emission in the SVG version as possible, correct? E.g. 
![image](https://cloud.githubusercontent.com/assets/1548516/18998145/9b7b8a1c-8737-11e6-9d95-19466f55688b.png)